charts/ccd-definition-store-api/values.aat.template.yaml

```diff
@@ -2,4 +2,14 @@ java:
   # Don't modify below here
```

Comment:
this is not the correct file. This is the AAT file. You have to change `values.yaml`

Reply:
The changes were supposed to  done for performance test only . But Do we want perfTest env to be exactly the same as the default  values.yml?  If it is the case I am only going to change the pool connection in the value.yml to the last known configuration 16  from /infrastructure